ci/travis-gap.sh

```diff
@@ -65,15 +68,23 @@ make
 cd ..
 
 # Orb
+ORB=orb-$ORBVERS
 echo "Downloading $ORB..."
-curl -O https://www.gap-system.org/pub/gap/gap4/tar.gz/packages/$ORB.tar.gz
+curl -L -O https://github.com/gap-packages/orb/releases/download/v$ORBVERS/$ORB.tar.gz
 tar xzf $ORB.tar.gz
 rm $ORB.tar.gz
 cd $ORB
 ./configure $PKG_FLAGS
 make
 cd ..
 
+################################################################################
+# Install required GAP packages
+
+curl -L -O http://www.gap-system.org/pub/gap/gap4pkgs/packages-required-master.tar.gz
```

Comment:
@james-d-mitchell Note that this gives you GAPDoc, at the moment version 1.6.1, in addition to 1.6 downloaded above.

Reply:
Thanks @alex-konovalov well spotted :)